packages/calypso-e2e/src/jest-playwright-config/environment.ts

```diff
@@ -248,16 +248,7 @@ class JestEnvironmentPlaywright extends JestEnvironmentNode {
 					}
 				}
 
-				// Delete recorded videos from tmp folder and close all contexts.
-				for await ( const context of contexts ) {
-					for await ( const page of context.pages() ) {
-						await page.close();
-						await page.video()?.delete();
-					}
-					await context.close();
```

Comment:
Deleting this section is the solution to the hang.

Reply:
I found that TeamCity has sever administrator options for cleaning up artifacts so this question isn't a concern.